src/test/java/org/broadinstitute/hellbender/tools/walkers/annotator/allelespecific/ReducibleAnnotationBaseTest.java

```diff
@@ -0,0 +1,104 @@
+package org.broadinstitute.hellbender.tools.walkers.annotator.allelespecific;
+
+import htsjdk.variant.variantcontext.VariantContext;
+import htsjdk.variant.variantcontext.VariantContextBuilder;
+import org.broadinstitute.hellbender.engine.FeatureDataSource;
+import org.broadinstitute.hellbender.tools.walkers.ReferenceConfidenceVariantContextMerger;
+import org.broadinstitute.hellbender.tools.walkers.annotator.VariantAnnotatorEngine;
+import org.broadinstitute.hellbender.tools.walkers.genotyper.*;
+import org.broadinstitute.hellbender.tools.walkers.genotyper.afcalc.GeneralPloidyFailOverAFCalculatorProvider;
+import org.broadinstitute.hellbender.utils.SimpleInterval;
+import org.broadinstitute.hellbender.utils.genotyper.IndexedSampleList;
+import org.broadinstitute.hellbender.utils.test.BaseTest;
+import org.broadinstitute.hellbender.utils.test.VariantContextTestUtils;
+import org.broadinstitute.hellbender.utils.variant.GATKVariantContextUtils;
+import org.testng.Assert;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+/**
+ * A class for storing end-end integration tests over artificial data intended to test allele specific annotation implementations,
+ * As of 8/29/17, test output was matched against GATK3 implementations of combineGVCFs().
+ */
+public abstract class ReducibleAnnotationBaseTest extends BaseTest {
```

Comment:
Can you confirm that all AS annotations present in GATK4 have a test that extends `ReducibleAnnotationBaseTest`?

Reply:
All of the concrete implementations do